### PR TITLE
feat(hardfork): add a new field "hardfork_features" to the return of RPC method "get_consensus"

### PR DESCRIPTION
**Pull request by yangby-cryptape**

### CODE REVIEW

File: rpc/README.md [hunk 1]
```diff
@@ -74,6 +74,8 @@ The crate `ckb-rpc`'s minimum supported rustc version is 1.51.0.
         * [Method `get_raw_tx_pool`](#method-get_raw_tx_pool)
     * [Module Stats](#module-stats)
         * [Method `get_blockchain_info`](#method-get_blockchain_info)
+        * [Method `get_features`](#method-get_features)
```
Comment (doitian) [inline rpc/README.md:77]: Better add to module chain.

Comment (yangby-cryptape) [reply rpc/README.md:77]: 👌

**Review by doitian — APPROVED** [review]

**Comment by quake** [comment]
I think these two rpc should be merged into `get_consensus` rpc.

**Review by quake — DISMISSED** [review]

File: rpc/README.md [hunk 2]
```diff
@@ -3512,6 +3523,19 @@ In JSONRPC, it is encoded as a 0x-prefixed hex string.
...
+
+The information about one hardfork feature.
+
+#### Fields
+
+`HardForkFeature` is a JSON object with the following fields.
+
+*   `rfc`: `string` - The related RFC ID.
+
+*   `epoch`: [`EpochNumber`](#type-epochnumber) `|` `null` - The first epoch when the feature is enabled.
```
Comment (doitian) [inline rpc/README.md:3536]: Please explain what does `null` mean.

**Review by doitian — CHANGES_REQUESTED** [review]

File: rpc/README.md (hunk 2 above)
Comment (keroro520) [reply rpc/README.md:3540]: previously I misunderstood *epoch* as `EpochNumberWithFraction`, while it should actually be interpreted as `EpochNumber`. And sadly the underlying types of  `EpochNumberWithFraction` and `EpochNumber` are both `u64`.

I would suggest changing the name to *epoch_number*, to reduce ambiguity.

File: rpc/README.md [hunk 3]
```diff
@@ -1260,6 +1261,10 @@ The response looks like below when `verbosity` is 0.
 
 Return various consensus parameters.
 
+##### Returns
+
+If any hardfork feature has `epoch=null`, it means the feature wouldn't be enabled forever.
```
Comment (doitian) [inline rpc/README.md:1266]:
```suggestion
If any hardfork feature has `epoch=null`, it means the feature will never be activated.
```

**Review by keroro520 — APPROVED** [review]

**Review by quake — APPROVED** [review]

**Review by doitian — APPROVED** [review]

**Comment by doitian** [comment]
bors r=quake,doitian,keroro520

**Pull request merged**